src/Client.php

```diff
@@ -25,295 +26,233 @@ class Client
-        return true;
+        var_dump($data);
+        $ids = array();
+        foreach ($data as $membership)
+            $ids[] = $membership["member"];
+        $data = $this->request("GET", "items/Members", ["data" => $ids]);
+        return $this->mapMembersToArray($data);
     }
 
-    public function getMembershipsByID($id)
-    {
-        $response = $this->guzzleClient->request('GET', 'api/v2/model', [
-          'query' => ['model_name' => 'person_model', 'method_name' => "get_valid_memberships_by_person_id", "person_id" => (string) $id]
-      ]);
-        $body = json_decode($response->getBody(), true);
-        $this->checkForError($body);
-        $ret = [];
-        foreach ($body as $membership) {
-            $ret[] = [
-            "active" => $membership['active'],
-            "name" => $membership['name'],
-            "fee" => $membership['fee'],
-            "issue_date" => $membership['issue_date'],
-            "expiry_date" => $membership['expiry_date'],
-            "is_general_membership" => $membership['is_general_membership']
-          ];
+    private function mapMembersToArray($members) : array {
+        $result = array();
+        foreach ($members as $person) {
+            $result[] = [
+                "id" => $person["id"],
+                "active" => true,  // TODO: This is no longer present in Directus
+                "first_name" => $person["first_name"],
+                "infix" => $person["infix"],
+                "last_name" => $person["last_name"],
+                "initials" => "", // TODO: Add this to Directus
+                "ssc_number" => $person["dms_id"]
+            ];
         }
-        return $ret;
+        return $result;
     }
 
-    public function getPayableMembershipsByID($id)
+    public function createPerson($values) : array
     {
-        $response = $this->guzzleClient->request('GET', 'api/v2/model', [
-          'query' => ['model_name' => 'person_model', 'method_name' => "get_payable_memberships", "person_id" => (string) $id]
-      ]);
-        $body = json_decode($response->getBody(), true);
-        $this->checkForError($body);
-
-        $ret = [];
-        foreach ($body as $membership) {
-            $ret[] = [
-            "active" => $membership['active'],
-            "name" => $membership['name'],
-            "fee" => $membership['fee'],
-            "issue_date" => $membership['issue_date'],
-            "expiry_date" => $membership['expiry_date'],
-            "is_general_membership" => $membership['is_general_membership']
-          ];
-        }
-        return $ret;
+        $address_data = [
+            "street" => $values["address_street"],
+            "number" => $values["address_number"],
+            "post_code" => $values["address_zip"],
+            "city" => $values["address_city"],
+            "country" => $values["address_country"],
+        ];
+        $address = $this->request("POST", "items/MemberAddresses", $address_data);
```

Comment:
I don't know if this is over-normalization in tables in this case, I would just leave it in the same table for clarity.